CHANGELOG-0.x.md

```diff
@@ -1,3 +1,11 @@
+# v0.8.0
+
+### Misc.
+* Release 0.7.0 part 3/3 ([#224](https://github.com/kubernetes-sigs/aws-fsx-csi-driver/pull/224), [@khoang98](https://github.com/khoang98))
```

Comment:
I would remove these but at the same time I don't mind keeping the list exhaustive and just keeping this stuff under 'Misc.' either

Reply:
Ah ok makes sense. Will leave for now but if it gets too unreadable, can remove these in future